### PR TITLE
146 - Style photo upload button for carriers

### LINKED ISSUE #146: Need styling fix on upload carrier photo
**Describe the bug**
When you add a new carrier at /carrier/new the "Browse" button needs styling, it doesn't fit inside the Photo box.   
Babywearing

### PR DESCRIPTION
**Pull request by Rgaliant**
Resolves #146   

### Description

added some CSS styling to the photo uploader for carriers

### Type of change

* New feature (non-breaking change which adds functionality)

### How Has This Been Tested?
create or edit carrier, photo uploader should work as normal

### Screenshots
Screen Shot 2019-10-16 at 7 09 33 PM

### CODE REVIEW

File: app/views/carriers/_carrier_form.html.erb [hunk 1]
```diff
@@ -57,9 +57,16 @@
...
   </div>
 
+  <div class="label">
+    Photos
+  </div>
   <div class="form-group">
-    <%= form.label :photos %>
-    <%= form.file_field :photos, multiple: true, class: "form-control" %>
+    <label class="btn btn-light">
+    Upload Image
```
Comment (dixpac) [inline app/views/carriers/_carrier_form.html.erb:65]: Oops...indentation

**Review by dixpac — CHANGES_REQUESTED** [review]
Thanks! Looks great, can you just fix the indentation

**Comment by pollygee** [comment]
Someone else got a PR in before this one, but they didn't add screenshots so we'll keep this one around in case we don't get their screenshots.

We have some more issues going in today @Rgaliant!  If you want I can tag you in the issues to see if you want to be assigned to them before we open them up.

**Comment by pollygee** [comment]
We merged the other PR, so closing this now.  I sent you a message on slack about some work coming today.  lmk

**Pull request closed without merging**